.gitlab-ci.yml

```diff
@@ -21,9 +22,18 @@ image: skilldlabs/php:73
     - branches
 
 .only_var_theme: &only_var_theme
+  cache:
+    key:
+      files:
+      # Until https://gitlab.com/gitlab-org/gitlab/issues/118466 lands ...
+        - web/themes/custom/XXX/package.json # ... this path has to be hardcoded
+        - web/themes/custom/XXX/yarn.lock # ... this path has to be hardcoded
+    paths:
+      - ${THEME_PATH}/${YARN_CACHE_FOLDER}/ # Populated during yarn install
   only:
     variables:
       - $THEME_PATH
+      - $YARN_CACHE_FOLDER
```

Comment:
This line should be removed imho, cause front jobs should be triggered only by theme defined, and nothing else (currently front job will try to run even if theme is not initialized yet, because YARN_CACHE_FOLDER is defined)